### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by Zzull**

### CODE REVIEW

File: src/ocsigenserver.ml [hunk 1]
```diff
@@ -18,6 +20,9 @@ let () =
       "--pidfile", Arg.String Ocsigen_config.set_pidfile, pid_msg;
       "-v", Arg.Unit Ocsigen_config.set_verbose, verbose_msg;
       "--verbose", Arg.Unit Ocsigen_config.set_verbose, verbose_msg;
+      "-vv", Arg.Unit Ocsigen_config.set_veryverbose, veryverbose_msg;
+      "-vvv", Arg.Unit Ocsigen_config.set_debug, debug_msg;
```
Comment (vouillon) [inline src/ocsigenserver.ml:24]: Add also `--veryverbose`?

File: src/Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 include ../Makefile.config
 
 all: confs
-	dune build
+	dune build --root ..
```
Comment (vouillon) [inline src/Makefile]: With this option, it seems `dune` tries to compile all dune projects in my home directory!

Comment (vouillon) [reply src/Makefile]:
```
Entering directory '/home/jerome'
Error: Too many opam files for package "dune-private-libs":
- dune.2.7.1/dune-private-libs.opam
- dune-merlin/dune-private-libs.opam
```

**Comment by Zzull** [comment]
Rebased. (Removed the `--root ..` commit and restored the `--veryverbose` option

**Pull request merged**